### PR TITLE
Backport/fix deleted users

### PR DESCRIPTION
**Pull request by eliegaboriau**
This PR cleans the module (using [this PR](https://github.com/OpenSourcePolitics/decidim-module-cleaner/pull/6) end [this PR](https://github.com/OpenSourcePolitics/decidim-module-cleaner/pull/9)) and backport the fix from [this PR](https://github.com/OpenSourcePolitics/decidim-module-cleaner/pull/9)

### CODE REVIEW

**Review by armandfardeau — APPROVED** [review]

**Comment by paulinebessoles** [comment]
@armandfardeau could you wait for all the reviews requested before merging?

**Pull request merged**
